.gitignore

```diff
@@ -42,6 +42,7 @@
 /vendor/llvm-build/
 /vendor/llvm/
 /vendor/node/deps/node-inspect/.npmrc
+/vendor/node/deps/npm/test/fixtures/config/.npmrc
```

Comment:
I don't think it is good idea to add files from submodules to `.gitignore`,
but since `check-for-leaks` can't ignore them, we have no choice =/

cc @zeke

Reply:
@alexeykuzmin Gah... I missed the spec/package-lock.json file.  I added it to .gitignore

Reply:
@alexeykuzmin is that annoying? If so, should we figure out a different approach?

Reply:
Can't we wild card this to like `**/.npmrc`, then that would catch :allthethings:

Reply:
@MarshallOfSound great idea!